projects/plugins/wpcomsh/private-site/private-site.php

```diff
@@ -770,6 +770,24 @@ function private_robots_txt() {
 	return "User-agent: *\nDisallow: /\n";
 }
 
+/**
+ * Renames the "Subscriber" role to "Viewer".
+ *
+ * @param \WP_Roles $roles WP_Roles object.
+ */
+function rename_subscriber_role_to_viewer( $roles ) {
+
+	// Dummy gettext calls to get strings in the catalog.
+	/* translators: User role. */
+	_x( 'Viewer', 'User role', 'wpcomsh' );
+
+	if ( site_is_private() && isset( $roles->roles['subscriber'] ) ) {
+		$roles->roles['subscriber']['name'] = 'Viewer';
+		$roles->role_names['subscriber']    = 'Viewer';
+	}
```

Comment:
Since `Viewer` is still not translated, I verified that this would return a translation by temporary switching to a different role that is already translated:


```suggestion
	// Dummy gettext calls to get strings in the catalog.
	/* translators: User role. */
	_x( 'Contributor', 'User role' );

	if ( site_is_private() && isset( $roles->roles['subscriber'] ) ) {
		$roles->roles['subscriber']['name'] = 'Contributor';
		$roles->role_names['subscriber']    = 'Contributor';
	}
```

<img width="1105" height="435" alt="Image" src="https://github.com/user-attachments/assets/63414c9f-95c9-46f4-9ff6-db8734ceb1b6" />